src/nvim/fold.c

```diff
@@ -762,7 +762,7 @@ void clearFolding(win_T *win)
  */
 void foldUpdate(win_T *wp, linenr_T top, linenr_T bot)
 {
-  if (compl_busy) {
+  if (State & INSERT) {
```

Comment:
It probably doesn't matter currently, but `compl_busy` could be used in later Vim patches to take on more meaning.